package.json

```diff
@@ -119,7 +119,6 @@
     "react-native-gesture-handler": ">=2.9.0",
     "react-native-keychain": ">=8.1.1",
     "react-native-markdown-display": ">=6.1.6",
-    "react-native-notifications": ">=4.3.3",
```

Comment:
Why should you remove this from peerDependencies? I'd imagine it still is a peer depend, no?

Reply:
yep restored it back to peerDependency